cedar-policy/src/api/err.rs

```diff
@@ -213,7 +213,11 @@ pub mod human_schema_errors {
     #[derive(Debug, Error, Diagnostic)]
     #[error(transparent)]
     #[diagnostic(transparent)]
-    pub struct ParseError(#[from] pub(super) cedar_policy_validator::HumanSyntaxParseError);
+    pub struct ParseError {
+        /// The errors reported by the parser
+        #[from]
+        pub errors: cedar_policy_validator::HumanSyntaxParseError,
```

Comment:
Is this error struct publicly exported?  If not, let's wrap in a struct at this layer (there are other examples of that pattern in this file)

Reply:
Not sure what you're referring to

Reply:
It is publicly exported, so this comment doesn't apply